yarn.lock

```diff
@@ -22537,6 +22537,11 @@ yn@3.1.1:
   resolved "https://registry.npmjs.org/yn/-/yn-3.1.1.tgz#1e87401a09d767c1d5eab26a6e4c185182d2eb50"
   integrity sha512-Ux4ygGWsu2c7isFWe8Yu1YluJmqVhxqK2cLXNQA5AcC3QfbGNpM7fu0Y8b/z16pXLnFxZYvWhd3fhBY9DLmC6Q==
 
+yn@^4.0.0:
```

Comment:
current version is 3.1.1 😅, if possible to bump the existing one I guess we could, ts-node is the culprit. Otherwise just keep both.